File: Programing_Basics_with_JS/conditionalStatementsMoreExercises/04.TransportPrice.js
function transportPrice(input) {
    let kilometers = Number(input[0]);
    let dayPeriod = input[1];

    let price = 0;

    if (kilometers < 20) {
        price += 0.70;
        if (dayPeriod === 'day') {
            price += kilometers * 0.79;
        } else if (dayPeriod === 'night') {
            price += kilometers * 0.90;
        }
    } else if (kilometers < 100) {
        price = kilometers * 0.09;
    } else {
        price = kilometers * 0.06;
    }
    
    console.log(price.toFixed(2));
}

transportPrice(['5', 'day'])
transportPrice(['7', 'night'])
transportPrice(['25', 'day'])
transportPrice(['180', 'night'])

File: Programming_Basics_with_Python/conditional_statements_more_exercise/04._transport_price.py
def transport_price(kilometers, time, taxi_first_fee, taxi_day_fee_per_km, taxi_night_fee_per_km, bus_minimum_km, train_minimum_km, bus_fee_per_km, train_fee_per_km):
    money = 0

    if kilometers >= train_minimum_km:
        money += kilometers * train_fee_per_km
    elif kilometers >= bus_minimum_km:
        money += kilometers * bus_fee_per_km
    else:
        money += taxi_first_fee
        if time == 'day':
            money += kilometers * taxi_day_fee_per_km
        elif time == 'night':
            money += kilometers * taxi_night_fee_per_km

    return f'{money:.2f}'


kilometers = int(input())
time = input()

taxi_first_fee = 0.70
taxi_day_fee_per_km = 0.79
taxi_night_fee_per_km = 0.90

bus_minimum_km = 20
bus_fee_per_km = 0.09

train_minimum_km = 100
train_fee_per_km = 0.06

print(transport_price(kilometers, time, taxi_first_fee, taxi_day_fee_per_km, taxi_night_fee_per_km, bus_minimum_km, train_minimum_km, bus_fee_per_km, train_fee_per_km))
